tools/appimage/make-appimage.sh

```diff
@@ -0,0 +1,79 @@
+#! /bin/bash
+
+set -e
+set -o pipefail
+
+show_usage_and_exit() {
+    echo "Usage: $0 <dir>"
+    exit 2
```

Comment:
Please say in the help that <dir> should be the result of... extracting the no-installer linux binary distribution?
Actually, I would like it better if the script would even take care of extracting that distribution from a downloaded file into a temp dir, and cleanup the mess at the end

Reply:
This issue seems broken, please revise.

Reply:
Please say in the help that `<dir>` should be the result of... extracting the no-installer linux binary distribution(?).

Actually, the script could even take care of extracting that distribution from a downloaded file into a temp dir, and cleanup the mess at the end.

Reply:
It could, but that doesn't make sense to me, really. It might make sense in your build infrastructure, though. I do not know how you plan to integrate this.

If you need such a script, you could add it as a second script which implements the downloading, extraction etc., and then calls this script. This script follows the Unix spirit, where there should be one tool for one job, and it should do that job well.

Reply:
Yes you're right, we should run this script at the same time we build the tarball.
What I don't understand is that there seem to be a usr/ hierarchy in the tarball, but you leave most stuff out of it. Shouldn't everything be moved inside usr/ ?

Reply:
Sorry for not replying in time. Feel free to move everything to a `usr/` subdirectory. It's common for AppImages, maybe even "best practice", but not a requirement technically. I'll send a follow-up PR.